.github/workflows/pushing_container_image.yml

```
@@ -0,0 +1,50 @@
+name: Publish image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registries:
+    name: Push image to multiple registries
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          buildkitd-flags: --debug
+      -
+        name: Login to ghcr.io
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: lowercase Repo
+        run: |
+            echo "REPO_OWNER=${GITHUB_REPO_OWNER,,}" >>${GITHUB_ENV}
+        env:
+          GITHUB_REPO_OWNER: ${{ github.repository_owner }}
+
+
+      - name: Build
+        run: |
+            cd docker
+            make REPO_OWNER=${{ env.REPO_OWNER }}
+     
+      - name: push images
+        run: |      
+            docker images
+            export TAG=$(git describe --always)
+            docker push ghcr.io/${{ env.REPO_OWNER }}/ub-12.04-base:$TAG
+            docker push ghcr.io/${{ env.REPO_OWNER }}/ub-16.04-base:$TAG
+            docker push ghcr.io/${{ env.REPO_OWNER }}/ub-18.04-base:$TAG
+            docker push ghcr.io/${{ env.REPO_OWNER }}/ub-12.04-conda:$TAG
```

```suggestion
            docker push ghcr.io/${{ env.REPO_OWNER }}/ub-16.04-base:$TAG
            docker push ghcr.io/${{ env.REPO_OWNER }}/ub-18.04-base:$TAG
            docker push ghcr.io/${{ env.REPO_OWNER }}/ub-16.04-conda:$TAG
```
Update following removal of Precise (12.04) #15